### ISSUE TITLE
TreeView : path is too long

### ISSUE THREAD

**Issue by PentiumRabbit** [alefragnani/vscode-bookmarks#96]
path is too long

**Comment by alefragnani** [comment]
Hi @PentiumRabbit ,

What do you mean?

**Comment by alefragnani** [comment]
Hi @PentiumRabbit ,
What do you mean?

**Comment by Fasani** [comment]
I think maybe @PentiumRabbit means this:
<img width="295" alt="screen shot 2018-05-09 at 12 30 00" src="https://user-images.githubusercontent.com/448410/39810376-13dc497c-5385-11e8-83db-7da2ddc93ff5.png">

Text could be right aligned so we can see the file name when the window is smaller.

**Comment by RsD0p9BK** [comment]
Yeah, there's missing ability to customize file path format (short, long, custom, etc)
